ProefProeveVeluwe/Assets/Scripts/Runtime/Animal Interaction/Petting/Petting.cs

```diff
@@ -6,39 +6,44 @@
 /// </summary>
 public class Petting : MonoBehaviour
 {
-    [SerializeField] private Camera MainCamera;
-    [SerializeField] private Slider HappinessBar;
+    // [SerializeField] private Slider HappinessBar;
```

Comment:
useless commenting

Reply:
it is there to visualize the progress bar but seeining as we don't have it yet I would like to keep it for when we do have it we can just uncomment it or is there something in the coding convention for this that I missed because then I switch it for that